core/controlplane/config/templates/cluster.yaml

```diff
@@ -1187,10 +1187,15 @@ addons:
 #   # certificate signing requests (csr) made to the API server using the bootstrap token. It's recommended to
 #   # also enable the rbac plugin in order to limit requests using the bootstrap token to only be able to make
 #   # requests related to certificate provisioning.
-#   # The bootstrap token must be defined in ./credentials/tokens.csv; kube-aws creates this file automatically
-#   # with a randomly generated token when this setting is enabled.
+#   # The bootstrap token is automatically generated in ./credentials/kubelet-tls-bootstrap-token.
 #   tlsBootstrap:
 #     enabled: true
+#     # If enabled, instructs the API server to enable the Node authorizer.
+#     # Only supported in Kubernetes 1.7.
+#     nodeAuthorizer: false
```

Comment:
Sorry for the delay.

These are unrelated features, but in the context of kube-aws, having TLS bootstrap enabled is a requirement for the Node authorizer to work because it expects nodes to use credentials that conform with the naming scheme (where username = `system:node:<node name>`, and group = `system:nodes`).